op-e2e/actions/proofs/isthmus_setcode_tx_test.go

```diff
@@ -0,0 +1,142 @@
+package proofs_test
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/core/vm/program"
+	"github.com/holiman/uint256"
+	"github.com/stretchr/testify/require"
+
+	batcherFlags "github.com/ethereum-optimism/optimism/op-batcher/flags"
+	actionsHelpers "github.com/ethereum-optimism/optimism/op-e2e/actions/helpers"
+	"github.com/ethereum-optimism/optimism/op-e2e/actions/proofs/helpers"
+)
+
+func TestSetCodeTxTypeIsthmus(gt *testing.T) {
+	t := actionsHelpers.NewDefaultTesting(gt)
+
+	var (
+		aa = common.HexToAddress("0x000000000000000000000000000000000000aaaa")
+		bb = common.HexToAddress("0x000000000000000000000000000000000000bbbb")
+	)
+
+	// hardcoded because it's not available until after we need it
+	bobAddr := common.HexToAddress("0x14dC79964da2C08b23698B3D3cc7Ca32193d9955")
+
+	// Create 2 contracts, (1) writes 42 to slot 42, (2) calls (1)
+	store42Program := program.New().Sstore(0x42, 0x42)
+	callBobProgram := program.New().Call(nil, bobAddr, 1, 0, 0, 0, 0)
+
+	alloc := *actionsHelpers.DefaultAlloc
+	alloc.L2Alloc = make(map[common.Address]types.Account)
+	alloc.L2Alloc[aa] = types.Account{
+		Code: store42Program.Bytes(),
+	}
+	alloc.L2Alloc[bb] = types.Account{
+		Code: callBobProgram.Bytes(),
+	}
+
+	testCfg := &helpers.TestCfg[interface{}]{
+		Hardfork: helpers.Isthmus,
+		Allocs:   &alloc,
+	}
+
+	tp := helpers.NewTestParams()
+	env := helpers.NewL2FaultProofEnv(t, testCfg, tp, &actionsHelpers.BatcherCfg{
+		MinL1TxSize:          0,
+		MaxL1TxSize:          128_000,
+		DataAvailabilityType: batcherFlags.CalldataType,
+	})
```

Comment:
you could just use `helpers.NewBatcherCfg()`

Reply:
nice, wasn't working before for some reason, but now it works